twine-upload.sh

```diff
@@ -40,6 +40,9 @@ if [[ ${INPUT_SKIP_EXISTING,,} != "false" ]] ; then
     TWINE_EXTRA_ARGS=--skip-existing
 fi
 
+if [[ ${VERBOSE,,} != "false" ]] ; then
```

Comment:
Apparently, this introduced a bug: https://github.com/pypa/gh-action-pypi-publish/issues/41.